lua/nvim-tree/actions/dispatch.lua

```diff
@@ -25,6 +25,7 @@ local Actions = {
   trash = require("nvim-tree.actions.fs.trash").fn,
   remove = require("nvim-tree.actions.fs.remove-file").fn,
   rename = require("nvim-tree.actions.fs.rename-file").fn(false),
+  relative_rename = require("nvim-tree.actions.fs.rename-file").fn(false, true),
```

Comment:
Please add API: `lua/nvim-tree/api.lua`

Reply:
ok - I've renamed action to rename_relative and used that name in the API since that naming matches the "rename_sub" pattern.

tested OK with `:lua require"nvim-tree.api".fs.rename_relative()`